helmfile/sampleContext.sh

```diff
@@ -10,7 +10,7 @@ export APP_LABEL_PROJECT_ID=16700
 export APP_LABEL_SECTION=fdasfda
 export APP_LABEL_BRANCH=IITB
 export DOCKER_TAG=latest
-export BRANCH=main
+export BRANCH=nightly
```

Comment:
This change doesn't appear to have anything to do with the logging feature being added.. (correct me if I'm wrong)